### PR TITLE
Fix documentation for cluster_role class resource

### PR DESCRIPTION
**Pull request by oharan2**

### CODE REVIEW

**Review by rnetser — APPROVED** [review]

File: ocp_resources/cluster_role.py [hunk 1]
```diff
@@ -4,8 +4,7 @@
 
 class ClusterRole(Resource):
     """
-    ClusterRole in kubernetes 'authorization-resources' official API:
-        https://kubernetes.io/docs/reference/kubernetes-api/authorization-resources/cluster-role-v1/
+    https://kubernetes.io/docs/reference/kubernetes-api/authorization-resources/cluster-role-v1/
```
Comment (myakove) [inline ocp_resources/cluster_role.py:7]: What was fixed here? this is the same link no?

Comment (oharan2) [reply ocp_resources/cluster_role.py:7]: In this discussion I was asked to remove the text above mentioning the API
https://github.com/RedHatQE/openshift-python-wrapper/pull/1353#discussion_r1301186544

Comment (rnetser) [reply ocp_resources/cluster_role.py:7]: so please fix the pr title

**Review by rnetser — APPROVED** [review]

**Review by rnetser — APPROVED** [review]

**Pull request merged**